src/components/Home/CommentModal.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div class="comment-box">
+    <fa-icon class="x" @click="close" icon="fa-solid fa-xmark fa-2xl" />
+    <input
+      id="author"
+      class="commentator"
+      placeholder="Seu nome"
+      v-model="comment.commentator"
+    />
+    <div>
+      <textarea class="content" v-model="comment.text" />
+    </div>
+    <button @click="addComment(post.comments, comment, post.id, post)">
+      Confirmar
+    </button>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Emit, Prop } from "vue-property-decorator";
+import commentService from "@/services/commentService";
+@Component({})
+export default class CommentModal extends Vue {
```

Comment:
Está usando algum componente?
```suggestion
import { Vue, Emit, Prop } from "vue-property-decorator";
import commentService from "@/services/commentService";

export default class CommentModal extends Vue {
```

Reply:
Não, usei pra indicar que este arquivo se trata de um componente, sem isso o componente não renderizava.